src/legacy/status_im/contact/block.cljs

```diff
@@ -45,16 +44,16 @@
     (apply
      rf/merge
      cofx
-     {:db                                                     (->
-                                                                db
-                                                                (update :chats dissoc public-key)
-                                                                (update :chats-home-list disj public-key)
-                                                                (assoc-in [:contacts/contacts public-key
-                                                                           :added?]
-                                                                          false))
-      :dispatch                                               [:shell/close-switcher-card public-key]
-      :effects/push-notifications-clear-message-notifications [public-key]}
-     (activity-center/notifications-fetch-unread-count)
+     {:db (->
+            db
+            (update :chats dissoc public-key)
+            (update :chats-home-list disj public-key)
+            (assoc-in [:contacts/contacts public-key
+                       :added?]
+                      false))
+      :dispatch [:shell/close-switcher-card public-key]
+      :effects/push-notifications-clear-message-notifications [public-key]
+      :fx [[:activity-center.notifications/fetch-unread-count]]}
```

Comment:
also im not sure about dispatch, sometimes it's in :fx sometimes not, probably should be always in :fx

Reply:
I've updated it but actually I wanted to keep changes to legacy namespaces to the minimum — this change was only because `:activity-center.notifications/fetch-unread-count` became an fx